etc/profile-m-z/mtpaint.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for mtpaint
+# Description: Simple painting and editing program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include mtpaint.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${PICTURES}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc 
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
```

Comment:
remove

Reply:
Why?

Reply:
It (can) cause graphic/performance issues in X11 depending on distro, xorg.conf, ...